app/views/posts/_form.html.erb

```diff
@@ -1,23 +1,5 @@
-<%= form_for [topic, post] do |f| %>
-<% if post.errors.any? %>
-    <div class="alert alert-danger">
-      <h4>There are <%= pluralize(post.errors.count, "error") %>.</h4>
-      <ul>
-        <% post.errors.full_messages.each do |msg| %>
-          <li><%= msg %></li>
-        <% end %>
-      </ul>
-    </div>
-  <% end %>
-  <%= form_group_tag(post.errors[:title]) do %>
-    <%= f.label :title %>
-    <%= f.text_field :title, class: 'form-control', placeholder: "Enter post title" %>
-  <% end %>
-  <%= form_group_tag(post.errors[:body]) do %>
-    <%= f.label :body %>
-    <%= f.text_area :body, rows: 8, class: 'form-control', placeholder: "Enter post body" %>
-  <% end %>
-  <div class="form-group">
-    <%= f.submit "Save", class: 'btn btn-success' %>
-  </div>
+<%= simple_form_for [topic, post] do |f| %>
+  <%= f.input :title, class: 'form-control', placeholder: "Enter post title" %>
```

Comment:
I notice you forgot to include the error display section that you had above on line 2. No need to re-do this or anything, I just wanted to point it out, and to show you how the GitHub inline comments work :-)

I can't merge your pull request because you haven't make me a 'collaborator' on this repository, but that's fine for now. We can worry about it later.